pkg/dao/rpm.go

```diff
@@ -170,3 +173,168 @@ func (r rpmDaoImpl) Search(orgID string, request api.SearchRpmRequest, limit int
 
 	return dataResponse, nil
 }
+
+const pagedRpmInsertsLimit = 5000
```

Comment:
mentioned in chat, but adding here too.  Maybe it makes sense for this to be a setting with some sensible default value.  That way we could adjust and change it more easily